phoenix-core/src/main/java/org/apache/phoenix/coprocessor/GroupedAggregateRegionObserver.java

```diff
@@ -402,8 +405,8 @@ private RegionScanner scanUnordered(ObserverContext<RegionCoprocessorEnvironment
             }
 
             Region region = c.getEnvironment().getRegion();
-            region.startRegionOperation();
             try {
+                region.startRegionOperation();
```

Comment:
Currently changed the code to throw stale region exception all when we get NSRE because hbase client may recreate the scanners with wrong region boundaries on NSRE. I have handling this in BaseScannerRegionObserver so need not handle in all observers.